### ISSUE TITLE
Unable to update Modules

### ISSUE THREAD

**Issue by gramundi** [microsoft/AzureAutomation-Account-Modules-Update#14]
Trying to update the Automation Account modules but it fails with :
Add-AzureRmAccount : Invalid provider type specified. At line:123 char:13 + Add-AzureRmAccount ` + ~~~~~~~~~~~~~~~~~~~~ + CategoryInfo : CloseError: (:) [Connect-AzureRmAccount], CryptographicException + FullyQualifiedErrorId : Microsoft.Azure.Commands.Profile.ConnectAzureRmAccountCommand

**Comment by AnatoliB** [comment]
Does your Automation account have an [Azure Run As account credential](https://docs.microsoft.com/azure/automation/manage-runas-account) created?

**Comment by gramundi** [comment]
Yep it has it is about an issue with the created connection. I have
attached a script to the case that I use to create the connection. It might
be something wrong on the script because the runbook cannot authenticate.
